.github/workflows/build-test-distribute.yaml

```diff
@@ -75,21 +116,6 @@ jobs:
             ${{ runner.os }}-devtools
       - name: Free up disk space for the Runner
         run: |
-          # source: https://github.com/apache/flink/blob/master/tools/azure-pipelines/free_disk_space.sh
```

Comment:
is it no longer needed? if so, why?

Reply:
Yes, it's no longer needed. I'll remove it. Thanks.